florist/api/db/entities.py

```diff
@@ -57,10 +57,10 @@ class Config:
         schema_extra = {
             "example": {
                 "client": "MNIST",
-                "service_address": "locahost:8081",
+                "service_address": "localhost:8001",
```

Comment:
Oh geez, thanks for fixing all the typos  😅

Reply:
No worries haha, I make lots of typos